docs/faq/go_env.md

```diff
@@ -19,7 +19,7 @@ This guide shows you how to perform the following:
 
 
 ### Step 1: Installing Go 
-The latest master branch of newt and newtmgr requires GO version 1.7.6 or higher. You can skip this step and proceed to Step 2 if you already have Go version 1.7.6 or higher installed.
+The latest master branch of newt and newtmgr requires GO version 1.10.3 or higher. You can skip this step and proceed to Step 2 if you already have Go version 1.10.3 or higher installed.
```

Comment:
I'm quite sure you could use `1.10` here instead of the specific release, they don't seem to add new APIs on point releases...